Source/WebKit/UIProcess/API/glib/IconDatabase.cpp

```diff
@@ -228,7 +231,7 @@ void IconDatabase::pruneTimerFired()
     }
 
     if (m_pruneIconsStatement->bindInt64(1, floor((WallTime::now() - notUsedIconExpirationTime).secondsSinceEpoch().seconds())) != SQLITE_OK) {
-        LOG_ERROR("FaviconDatabse::pruneTimerFired failed: %s", m_db.lastErrorMsg());
+        LOG_ERROR("FaviconDatabase::pruneTimerFired failed: %s", m_db.lastErrorMsg());
```

Comment:
```suggestion
        LOG_ERROR("IconDatabase::pruneTimerFired failed: %s", m_db.lastErrorMsg());
```

Do a Ctrl+H for this please; you fixed the typo, but the class name has been wrong since [190798@main](https://commits.webkit.org/190798@main) a decade ago.

Reply:
Good idea. Also the `LOG_ERROR` macro picks `__FILE__`, `__LINE__`, and `WTF_PRETTY_FUNCTION` so we don't really need to add the class and method name in the format strings. I'll remove them.